wandb/integration/openai/resolver.py

```diff
@@ -1,28 +1,36 @@
 import io
 import logging
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Sequence
 
 from wandb.sdk.data_types import trace_tree
-from wandb.sdk.integration_utils.llm import Response
+from wandb.sdk.integration_utils.auto_logging import Response
 
 logger = logging.getLogger(__name__)
 
 
 class OpenAIRequestResponseResolver:
```

Comment:
same as above Subclass the `ArgumentResponseResolver` here

Reply:
same

Reply:
Addressed in: https://github.com/wandb/wandb/pull/5535/commits/ff77a17c16fdb32592a81405d16cb3755b64ca12